src/components/Vega/EntropyPlot.js

```diff
@@ -29,8 +29,45 @@ const PlotContainer = styled.div``;
 
 const EntropyPlot = observer(({ width }) => {
   const vegaRef = useRef();
-  const { configStore, dataStore, UIStore, plotSettingsStore, snpDataStore } =
-    useStores();
+  const {
+    configStore,
+    dataStore,
+    UIStore,
+    plotSettingsStore,
+    snpDataStore,
+  } = useStores();
+
+  const nameToAbbr = {
```

Comment:
We should add these to the genes and proteins definitions file (inside static_data) as a new field, like "shortName" or something. We may need these later

Reply:
I've been using abbr (abbreviation) as a key but we can change it to shortName if that would be more descriptive.

Reply:
"abbr" is fine – I was just making the point that we should put these in each gene/protein object